pkg/bbgo/interact.go

```diff
@@ -21,6 +21,13 @@ type PositionReader interface {
 	CurrentPosition() *types.Position
 }
 
+type StrategyController interface {
+	SuspendStrategy(ctx context.Context) error
+	ResumeStrategy() error
+	GetStrategyStatus() bool
+	EmergencyStop(ctx context.Context) error
```

Comment:
Pull out `EmergencyStop` to a single instance, so that we can implement it separately.

Reply:
done